O/osslsigncode/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "osslsigncode"
+version = v"2.10.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/mtrojnar/osslsigncode",
+                  "4568c890cc1538ca80be3ee36775ba42223dea04"),
+]
+
+script = raw"""
+    cd $WORKSPACE/srcdir/osslsigncode
+
+    sed -i 's/Ws2_32/ws2_32/g' CMakeLists.txt
+
+    cmake -B build \
+        -DCMAKE_INSTALL_PREFIX=${prefix} \
+        -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DZLIB_INCLUDE_DIR=/workspace/destdir/include \
+        -DZLIB_LIBRARY=/workspace/destdir/lib/libz.a
+
+    cmake --build build --parallel ${nproc}
+    
+    install_license LICENSE.txt
+    install -Dvm 755 "build/osslsigncode${exeext}" "${bindir}/osslsigncode${exeext}"
+"""
+
+platforms = supported_platforms()
+filter(!(==)(Platform("i686", "windows")), platforms)
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("osslsigncode", :osslsigncode)
+]
+
+dependencies = [
+    Dependency("OpenSSL_jll", compat="3.0.16"),
+    Dependency("Zlib_jll", compat="1.3.1"),
```

Comment:
This won't work in any released version of Julia, either do
```suggestion
    Dependency("Zlib_jll"; compat="1.2.12"),
```
or
```suggestion
    Dependency("Zlib_jll"),
```

Reply:
Doesn't static linking solve this issue?

Reply:
You either do static linking ***or*** use zlib as a runtime dependency, doing both doesn't make sense: https://github.com/JuliaPackaging/Yggdrasil/pull/10950#discussion_r2041104429.  We usually try to do dynamic linking as much as possible though (although this is an executable, so the argument is a bit moot)